core/src/main/java/module-info.java

```diff
@@ -484,6 +484,7 @@
   exports org.mule.runtime.core.internal.util.mediatype to
       org.mule.runtime.extensions.support;
   exports org.mule.runtime.core.internal.util.message to
+      org.mule.runtime.core.components,
```

Comment:
I suppose if the imports were really unused, then this one is not needed either.